lib/matplotlib/tests/test_gridspec.py

```diff
@@ -1,5 +1,5 @@
 import matplotlib.gridspec as gridspec
-from nose.tools import assert_raises, assert_equal
+from numpy.testing import assert_raises, assert_equal
```

Comment:
yep.
FYI, if you look at skimage's pytest transition PR, you'll have a good idea of what will work and what won't.
scikit-image/scikit-image#2468